.github/workflows/ci.yml

```diff
@@ -21,3 +21,23 @@ jobs:
           coverage: xdebug
       - run: composer install
       - run: vendor/bin/phpunit --coverage-text
+
+  PHPStan:
+    name: PHPStan
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php: [ 8.1 ]
```

Comment:
Even though we only run PHPStan on a single PHP version currently, I've added a matrix anyway. IMHO it does not hurt and we can quickly add new versions if needed.

Reply:
Good call, makes sense 👍